syntheseus/tests/interface/test_models.py

```diff
@@ -14,10 +14,4 @@ def test_prediction():
     assert np.isclose(prediction.get_prob(), 0.5)
     assert np.isclose(prediction.get_log_prob(), math.log(0.5))
 
-    other_prediction = Prediction(input=Molecule("N"), output=Molecule("NC=O"), probability=0.5)
-    prediction_list = PredictionList(
-        input=Molecule("C"), predictions=[prediction, other_prediction]
-    )
-
-    assert prediction_list.predictions == [prediction, other_prediction]
-    assert prediction_list.truncated(num_results=1).predictions == [prediction]
+    Prediction(input=Molecule("N"), output=Molecule("NC=O"), probability=0.5)
```

Comment:
The asserts seemed redundant.

If the `PredictionList` is replaced by a list of predictions, then the first assert would essentially be:

```python
assert [prediction, other_prediction] == [prediction, other_prediction]
```

which is trivially true.


Secondly, because there is no `truncated()` method, the second assert would essentially be:

```python
assert [prediction, other_prediction][:1] == [prediction]
```

This also seemed trivially true. Therefore removing the asserts seemed like the most sensible option to me. This does not mean the test is vacuous though: it still checks that the data structures can be instantiated. Does this seem fine to you @kmaziarz?

Reply:
Ah, yes, makes sense then.